service/src/java/org/apache/hive/service/server/KillQueryImpl.java

```diff
@@ -116,6 +119,8 @@ public static void killChildYarnJobs(Configuration conf, String tag, String doAs
 
   private static boolean isAdmin() {
     boolean isAdmin = false;
+    // RANGER-1851
+    HivePrivilegeObject serviceNameObj = new HivePrivilegeObject(HivePrivilegeObject.HivePrivilegeObjectType.SERVICE_NAME, null, "hiveservice");
```

Comment:
Instead of hard-cording "hiveservice" value, have you thought about making this configurable?

Reply:
So, currently the ranger service expects only "hiveservice" Ref. https://issues.apache.org/jira/browse/RANGER-1851. This has been the case since '17. I am even thinking of using the same for abort transactions as well. Do you see any reason why it needs to be made configurable?